monitoring/snippets/v3/cloud-client/snippets_test.py

```diff
@@ -52,72 +52,56 @@ def write():
-    out, _ = capsys.readouterr()
-    assert "logging.googleapis.com/byte_count" in out
+def test_list_metric_descriptors(capsys) -> None:
+    result = snippets.list_metric_descriptors(PROJECT_ID)
+    assert any(item.type == "logging.googleapis.com/byte_count" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_resources(capsys):
-    snippets.list_monitored_resources(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "pubsub_topic" in out
+def test_list_resources(capsys) -> None:
+    result = snippets.list_monitored_resources(PROJECT_ID)
+    assert any(item.type == "pubsub_topic" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_get_resources(capsys):
-    snippets.get_monitored_resource_descriptor(PROJECT_ID, "pubsub_topic")
-    out, _ = capsys.readouterr()
-    assert "A topic in Google Cloud Pub/Sub" in out
+def test_get_resources(capsys) -> None:
+    result = snippets.get_monitored_resource_descriptor(PROJECT_ID, "pubsub_topic")
+    assert result.display_name == "Cloud Pub/Sub Topic"
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_time_series(capsys, write_time_series):
-    snippets.list_time_series(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "gce_instance" in out
+def test_list_time_series(capsys, write_time_series) -> None:
+    result = snippets.list_time_series(PROJECT_ID)
+    assert any(item.resource.type == "gce_instance" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_time_series_header(capsys, write_time_series):
-    snippets.list_time_series_header(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "gce_instance" in out
+def test_list_time_series_header(capsys, write_time_series) -> None:
+    result = snippets.list_time_series_header(PROJECT_ID)
+    assert any(item.resource.type == "gce_instance" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_time_series_aggregate(capsys, write_time_series):
-    snippets.list_time_series_aggregate(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "points" in out
-    assert "interval" in out
-    assert "start_time" in out
-    assert "end_time" in out
+def test_list_time_series_aggregate(capsys, write_time_series) -> None:
+    result = snippets.list_time_series_aggregate(PROJECT_ID)
+    assert any(item.resource.type == "gce_instance" for item in result)
 
 
 @backoff.on_exception(backoff.expo, (ServiceUnavailable), max_tries=3)
-def test_list_time_series_reduce(capsys, write_time_series):
-    snippets.list_time_series_reduce(PROJECT_ID)
-    out, _ = capsys.readouterr()
-    assert "points" in out
-    assert "interval" in out
-    assert "start_time" in out
-    assert "end_time" in out
+def test_list_time_series_reduce(capsys, write_time_series) -> None:
```

Comment:
ditto: We probably don't need `capsys` anymore.

Reply:
done